### ISSUE TITLE
Cheatsheet Recipe blocked on `recipegirl.com`

### ISSUE THREAD

**Issue by yasminyin96** [neevaco/neeva-ios#4290]
## Expected behavior

Show recipe widget.

## Steps to reproduce the problem

https://www.recipegirl.com/strawberry-chocolate-chip-bread-pudding/

1. Go to this website
2. Click `NeevaScope` button

## Screenshots or recordings

https://user-images.githubusercontent.com/104786010/186540815-bed15339-9fa1-4ffe-90eb-750c44bb0e61.mov

https://user-images.githubusercontent.com/104786010/186540947-c2ab4756-8557-4495-9b2f-95908e645d5e.MP4

## Hardware and software

- Your device: iPhone 13
- OS version: 15.5
- App version: Neeva Browser 1.55.3

**Comment by darinwf** [comment]
@EdwardJTL FYI

**Comment by EdwardJTL** [comment]
this is not being shown because it didnt pass our `DomainAllowList`. @macy-n should this one be blocked?

**Comment by yasminyin96** [comment]
> this is not being shown because it didnt pass our `DomainAllowList`. @macy-n should this one be blocked?

I see. I have a look at the `DomainAllowList` on IOS, there is no 'recipegirl.com'. I will add this `DomainAllowList` to Android. Thank you.

**Comment by EdwardJTL** [comment]
this will be either a "wont fix" or fix by #4292. closing this now :)

**Comment by dan-alcantara-neeva** [comment]
Shouldn't a DomainAllowList be managed by the backend?  If it's not allowed, we shouldn't be getting anything on the clients, no?

**Comment by dan-alcantara-neeva** [comment]
Re-opening for questions

**Comment by EdwardJTL** [comment]
i think the new ticket is a good place to have these follow up questions specifically about the use of the allow list.
I had also added this as a topic to our next weekly sync. We can ask the backend folks there.

**Comment by dan-alcantara-neeva** [comment]
When's the weekly sync @EdwardJTL?  Blocking https://github.com/neevaco/neeva-android/pull/919 until the discussion happens.

**Comment by EdwardJTL** [comment]
> When's the weekly sync @EdwardJTL? Blocking [neevaco/neeva-android#919](https://github.com/neevaco/neeva-android/pull/919) until the discussion happens.

mondays. I see that you are already invited on the calendar event
